### PR TITLE
#523 notification module integration

### PR DESCRIPTION
**Pull request by vivek-devtron**
# Description
To make CICD light we are breaking Notification module separately and listing it on the modules pages. So if user wants it he can install it.

Divided CICD into 2 part
CICD and Notification

Hide Notification menu from global config navigation

Fixes # [(issue)](https://github.com/devtron-labs/dashboard/issues/523)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- verified on the global config navigation, the notification option should only come when notifier module is installed

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/config/constants.ts [hunk 1]
```diff
@@ -188,8 +188,7 @@ export const AppConfigStatus = {
 }
 
 export const FullRoutes = {
-    LOGIN: `${Routes.LOGIN}`,
-    CENTRAL: 'https://api.devtron.ai',
+    LOGIN: `${Routes.LOGIN}`
```
Comment (sohel-rp) [inline src/config/constants.ts:191]: Use `Routes.LOGIN` directly

**Review by sohel-rp — APPROVED** [review]

File: src/components/login/login.service.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import { post, put, get } from '../../services/api';
-import { FullRoutes } from '../../config';
+import { Routes } from '../../config';
 
 export function getSSOConfigList(): Promise<any> {
     const URL = `sso/list`;
```
Comment (shivani170) [inline src/components/login/login.service.ts:5]: Can. you u=use constant here as well

Comment (vivek-devtron) [reply src/components/login/login.service.ts]: done

File: src/config/constants.ts (hunk 1 above)
Comment (vivek-devtron) [reply src/config/constants.ts]: done

**Review by shivani170 — APPROVED** [review]

**Pull request merged**